### PR TITLE
<Fixes#967> Fix button names in download notification

### LINKED ISSUE #967: fix button names in download notification
Instead of "Play", we should Probably have the string "Resume"

**Steps to reproduce the behavior:**
1. Download a zim file
2. Pause the download
3. Open notifications

**Screenshots**

screenshot_20190210-190737

**Environment**
- Version of Kiwix Android : 2.4 Build 55
- Device : Samsung S6
- OS version : Android 7.0

### PR DESCRIPTION
**Pull request by iadeelzafar**
### Changes:

* add Resume string value
* add this label to the download notification label instead of PLAY.

### CODE REVIEW

**Comment by iadeelzafar** [comment]
@abdulwd I had re-forked the project, that's why I'm creating a new pull request for same issue. Sorry about that. Kindly review this now.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -96,6 +96,7 @@
   <string name="download_play">Play</string>
```
Comment (abdulwd) [inline app/src/main/res/values/strings.xml]: Right-click `download_play` and select `Find Usages`, you will find that there are none so can delete this line.

Comment (iadeelzafar) [reply app/src/main/res/values/strings.xml]: @abdulwd done.

**Comment by soloturn** [comment]
@iadeelzafar , not sure why you reforked - am assuming your branches got out of sync with upstream. here some hints how to keep it up to date: https://help.github.com/articles/syncing-a-fork/. personally i prefer rebase  / fast-forward merge to merge used in the gihub example as it makes it easier for the receiving repo. like this:
```
git fetch -p --all
git checkout <yourbranch>
git rebase upstream/<yourbranch>
git push -f
```

**Comment by iadeelzafar** [comment]
@mhutti1 I think this PR is ready to be committed.

**Comment by iadeelzafar** [comment]
@mhutti1 I think this PR is ready to be committed.

**Pull request merged**